Updated material based on feedback received (#3827)
galaxyproject/training-material: topics/contributing/tutorials/learning-principles/tutorial.md

* Understanding  how people learn makes you a more aware and therefore better teacher
* You  can use such knowledge to improve your day-to-day teaching

> Ausubel, quoted by {% cite Dnarte2011 %}, indicates that meaningful learning generally arises from a process of learning to learn. According him, ff meaningful learning is achieved, it transcends the rote repetition of unconnected content and allows for the construction of meaning, making sense of what has been learned, and understanding its scope of application and relevance in academic and everyday situations.

Reviewer: ```suggestion
David Ausubel indicates that meaningful learning generally arises from a process of learning to learn. According him, if meaningful learning is achieved, it transcends the rote repetition of unconnected content and allows for the construction of meaning, making sense of what has been learned, and understanding its scope of application and relevance in academic and everyday situations.
```

>
> The **Information Processing Theory** describes learning as the result of sequential processing of information, which involve three types of memory: immediate memory, working memory and long term memory [33]. According to this theory, adequate learning resources require to assume two key elements: firstly, the fact that working memory is limited; and secondly, that the interaction between working and long term memory plays an important role in the construction and transferability of knowledge [6] [13] [34].
>
> On the other hand, **Cognitive Load Theory** seeks specifically to address the efficiency with which information is processed. It states that the verbal and visual information is processed by independent cognitive structures, both of them with a limited capacity [35] [36] [37] [38] [39]. An interesting concept derived from this theory is the cognitive overload, which refers to those situations in which the information flow exceeds the learner’s processing capacity, resulting in an inhibition of the learning process [40] [41].
>
>
> #### Constructivist Learning Theory
>

> > **Constructivism** holds that learning is essentially active; a person who learns something new incorporates it into their previous experiences and their own mental structures. Each new piece of
> information is assimilated and deposited into a network of previously existing knowledge and experiences  ({% cite Abbot1999 %}). All these ideas have been taken from different nuances, and two authors stand out: Jean Piaget with **psychological constructivism** and Lev Vigotsky with **social constructivism**.

Reviewer: ```suggestion
> **Constructivism** holds that learning is essentially active; a person who learns something new incorporates it into their previous experiences and their own mental structures. Each new piece of information is assimilated and deposited into a network of previously existing knowledge and experiences  ({% cite Abbot1999 %}). All these ideas have been taken from different nuances, and two authors stand out: Jean Piaget with **psychological constructivism** and Lev Vigotsky with **social constructivism**.
```

    Example: A competent practitioner in a Carpentries workshop might have used the shell before and understand how to move around directories and use individual programs, but they might not understand how they can fit these programs together to build scripts and automate large tasks.

- **Expert**: someone who can easily handle situations that are out of the ordinary.

    Example: An expert in a Carpentries workshop may have experience writing and running shell scripts and, when presented with a problem, immediately sees how these skills can be used to solve the problem.

> <details-title>The Carpentries</details-title>
>
> [The Carpentries](https://carpentries.org/) was started by Greg Wilson and Brent Gorda and they coined the term "The Carpentries" for enabling researchers to do "computational thinking" and use tools of computation effectively.
>
> The Style of the Carpentries are that teachers are where the learners are e.g. by doing live-coding together.
{: .details}

> #  Cognitivist Learning Theory: Memory and learning

Reviewer: ```suggestion
##  Cognitivist Learning Theory: Memory and learning
```

    - Design instruction and learning experiences i.e. how you are to teach/train, which infrastructure and how to present the materials etc
    - Assess learning (taking place during curricula or course and after)
    - Align instruction with learners' level of complexity of thinking as well as experience.

> # Critical Learning Theory: The Hidden Curriculum

Reviewer: ```suggestion
## Critical Learning Theory: The Hidden Curriculum
```

Reviewer: That will fix the linting problem.

The mental model of an expert in any given subject will be far larger and more complex than that of a novice, including both more concepts and more detailed and numerous relationships. Linking with Dreyfus model:

- A novice has a minimal mental model of surface features of the domain. Inaccuracies based on limited prior knowledge may interfere with adding new information. Predictions are likely to borrow heavily from mental models of other domains which seem superficially similar.
- A competent practitioner has a mental model that is useful for everyday purposes. Most new information they are likely to encounter will fit well with their existing model. Even though many potential elements of their mental model may still be missing or wrong, predictions about their area of work are usually accurate.
- An expert has a densely populated and connected mental model that is especially good for problem solving. They quickly connect concepts that others may not see as being related. They may have difficulty explaining how they are thinking in ways that do not rely on other features unique to their own mental model.

> ### Misconceptions

Reviewer: ```suggestion
## Misconceptions
```
